docs/user/getting-started.md

```diff
@@ -40,9 +39,8 @@ local-path-storage   local-path-provisioner-7577fdbbfb-wxbck      1/1     Runnin
 Verify Helm is installed and can access the cluster:
 
 ```console
-$ helm list
```

Comment:
the commands the do not have any output in the `console`, i've removed the `$`sign. It makes it much easier to copy.